lib/IMAP/Sync/Synchronizer.php

```diff
@@ -79,6 +81,11 @@ public function sync(Horde_Imap_Client_Base $imapClient,
 				throw new UidValidityChangedException();
 			}
 			throw $e;
+		} catch (Horde_Imap_Client_Exception $e) {
+			if ($e->getCode() === Horde_Imap_Client_Exception::MBOXNOMODSEQ) {
+				throw new MailboxDoesNotSupportModSequencesException();
```

Comment:
please include the previous (inner) exception by passing it as third argument of `MailboxDoesNotSupportModSequencesException`

Reply:
Done